flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/KafkaConsumerTestBase.java

```diff
@@ -257,7 +258,7 @@ public void run() {
 		runner.join();
 
 		final Throwable t = errorRef.get();
-		if (t != null) {
+		if (t != null && ((JobExecutionException) t).getStatus() == ApplicationStatus.FAILED) {
```

Comment:
Why is this change necessary?

Reply:
@tillrohrmann , because this change checks whether the application status is FAILED. Only when application status is FAILED, this could throw Exception with fail message.

Reply:
I think if program throws a `JobCancellationException`, then we don't set it in the `errorRef` variable. Hence, this test should not be necessary.

Reply:
@tillrohrmann , I got your point for this change, and I would like to update commit for this.